tests/test_csc.py

```diff
@@ -157,7 +157,9 @@ async def test_start_scan_dt(self):
                 unittest.mock.AsyncMock(return_value=([2], ["EM1_O_20221130_000002"]))
             )
             await self.remote.cmd_startScanDt.set_start(scanDuration=2)
-            await self.remote.evt_largeFileObjectAvailable.next(flush=False, timeout=10)
+
+            # await self.remote.evt_largeFileObjectAvailable.next(flush=False,
+            #                                                     timeout=10)
```

Comment:
Is there a reason that this line is commented out?

Reply:
Not sure. I think it was needed to be able to pass the tests...

Reply:
We need to check that this event is published.